x-pack/plugins/security_solution/server/lib/detection_engine/scripts/roles_users/README.md

```diff
@@ -0,0 +1,13 @@
+1. When first starting up elastic, detections will not be available until you visit the page with a SOC Manager role or Platform Engineer role
+2. The T1 and T2 Analyst roles do not seem to have much of a functional difference but I created them as is.
```

Comment:
It's just the value-list difference right? `T1 Analysts` can't view any existing value lists, but `T2` can?

Reply:
Ah I should update this description. I updated the table but not the notes I had written here.

Reply:
Ah I should update this description. I updated the table but not the notes I had written here.